pkg/subgraph/subgraph.yaml

```diff
@@ -6,15 +6,15 @@ schema:
 dataSources:
   - kind: ethereum/contract
     name: RegistryFactory
-    network: localhost
+    network: arbitrum-sepolia
```

Comment:
those changes are not going to affect the localhost env?

Reply:
should affect but because people run local command that its overriden.